### DISCUSSION TITLE
Labs Install Help on Hero 11

### DISCUSSION THREAD

**Discussion by overossm** [gopro/labs#919] [Q&A]
Hello! I installed the GoPro labs firmware on the micro SD card and followed the instructions to insert it into the GoPro camera while it is turned off and then turn it on. 

The instructions said the camera would turn on and off multiple times, and a checkmark would appear. However, it did not turn on and off, and no check mark appeared. 

I verified that the firmware is installed on the SD card, but I haven't had any luck getting a check mark or any sort of verification that the camera is recognizing it. 

Any advice? 

I appreciate any help you can provide.

**Comment by dnewman-gpsw** [comment]
Lots of YouTube videos to show a GoPro Labs install.  Maybe it installed fine, a blink as you could have missed the update complete message, so check the version number in the camera's menu.

**Comment by overossm** [comment]
Thank you for your response.
After watching an installation YouTube video, I figured out the issue!
Apparently, it was the second time I downloaded the firmware, so my
computer auto-saved the folder as "Update 2," which I didn't think much of
until a few minutes ago. I changed the folder's name to just "Update," and
retried installation, which was a success! Huzzah!
